containers/unit-tests/Dockerfile

```diff
@@ -13,6 +13,9 @@ WORKDIR /home/builder
 
 ENV LANG=C.UTF-8
 
+# HACK: unbreak distcheck on Debian: https://bugs.debian.org/1035546
+ENV DEB_PYTHON_INSTALL_LAYOUT=deb
```

Comment:
See https://salsa.debian.org/cpython-team/python3/-/blob/master/debian/patches/sysconfig-debian-schemes.diff#L56

Reply:
If I've wrapped my head around this situation properly then it also means that something like this, for example:

./configure
make
sudo make install

would result in Cockpit getting installed into `/usr/local/local/` on Debian ...

Reply:
The crux of the issue is that although sysconfig consults has a check for situations where the prefix is not the same as the base prefix (ie: venvs) it doesn't consider pip's `--prefix` flag when applying this logic...

Reply:
notes for myself:
 - check the `sudo make install` behaviour from above
 - possibly: toss away `pip install` and install the wheel ourselves; create single-line wrappers for cockpit-{bridge,askpass,beiboot} in our tree and install via scripts directory; use `#!/usr/bin/env python3` hashbangs

Reply:
Indeed a `make install DESTDIR=/tmp/i` in Debian creates /tmp/i/usr/local/local/lib/python3.11, and /tmp/i/usr/local/local/bin/cockpit-{bridge,askpass,beiboot} as we suspected. 

I tried to convert to python3-installer:
```diff
--- src/Makefile.am
+++ src/Makefile.am
@@ -26,7 +26,7 @@ install-python:
        @# wheel-based installation with .dist-info.
        @# This needs to work on RHEL8 up through modern Fedora, offline, with
        @# system packages available to the build.
-       python3 -m pip install --no-index --force-reinstall --root='$(DESTDIR)/' --prefix='$(prefix)' \
+       python3 -m installer --destdir='$(DESTDIR)/' --prefix='$(prefix)' \
                "$$(python3 '$(srcdir)'/src/build_backend.py --wheel '$(srcdir)' tmp/wheel)"
        mkdir -p $(DESTDIR)$(libexecdir)
        mv -t $(DESTDIR)$(libexecdir) $(DESTDIR)$(bindir)/cockpit-askpass $(DESTDIR)$(bindir)/cockpit-beiboot
```

and it has the same problem. That's unsurprising, as the [offending patch](https://salsa.debian.org/cpython-team/python3/-/blob/master/debian/patches/sysconfig-debian-schemes.diff) is in Debian's cpython, not in pip.